docs/runbook/Features.md

```diff
@@ -173,90 +98,168 @@ on the rooftop is selected, which can be difficult to do manually.
-[Pop Placement](#pop-placement) feature for information on how to quickly place
-extra POPs.
-
-**How to use it?**
-
-1.  Go to the Network Design section.
-2.  Set the Maximize minimum guaranteed bandwidth parameter to "True".
-3.  Set the Link Capacity Filter value (Gbps) as required; Default = 0
-
-**Note:** Some links might have 0 capacity. In that case, if such a link is
-necessary to reach the site, it\'s still unreachable. There is also a
-parameter to filter links below some capacity. That could be useful in some
-rare cases where low capacity links are connecting some sites causing the max
-common bandwidth to be low. In that case, it's better to disconnect those sites
-by filtering out the low capacity links (that is, prevent the optimizer from
-selecting them). Regardless, if you choose to filter links below 10Mbps for
-example, then all such links will be marked unreachable.
-
-**Key Metrics**
-
-- Number of active CN/DN/POPs connected to demand sites
-- Number of active/connectable candidate/total candidate CN sites
-- Full topology flow results - Percent of demand sites, minimum guaranteed
-  bandwidth (Gbps)
+POP Placement allows you to specify additional POPs you would like to add to a
+candidate topology. The POPs are selected from the candidate DNs.
 
-## Tiered Service
+This can be useful to meet CIR requirements when the number of provided
+POPs is insufficient. It can also help connect disconnected portions of the
+network. It is particularlly useful in early-stage planning where not all
+possible POP locations are already known.
 
-Previously the TG Planner only provided identical bandwidth to each of the CNs
-it serves. With the Tiered Service feature, you can now provide different levels
-of bandwidth to a selected number of CNs. The Tiered Service feature properly
-addresses two use cases:
+**How to use it**
 
-1.  Multi-dwelling buildings with units requiring different levels of bandwidth.
-2.  Network shared by businesses and residential homes where businesses require
-    a higher amount of bandwidth.
+1. Specify the number of additional POPs you would like in `NUMBER_OF_EXTRA_POPS`
+   under the `NETWORK_DESIGN` section.
+
+## Demand Models
+
+Demand sites are imaginary sites in the network that are added to represent
+the final destination of downstream flow from the POPs. In graph theory
+terminology, they are the sinks nodes of the directed graph that represents the
+network. They are distinct from CNs because, in part,
+
+- Not all networks will have a CN
+- Multiple CNs can connect to the same demand site allowing the network to
+  decide which CN is needed
+- A DN and a CN can connect to the same demand site allowing the network to
+  decide which one is needed (e.g., a DN can both serve the customer like a
+  CN while simultaneously pushing data to other customers downstream).
+- Each DN and CN can connect to a different number of demand sites resulting
```

Comment:
Are you suggesting to say "Each site can connect..."

Reply:
No. I don't know if this makes sense, but let me try to explain what I was thinking. It seems like every DN/CN has a different number of sites it can connect to which means every DN/CN has its own specific demand requirements, but nothing general to all DNs or CNs (i.e., each DN/CN is different but this by itself doesn't seem to suggest that it contributes to the "class of DNs" being different from the "class of CNs").